### DISCUSSION TITLE
Export blendShape without Mesh !

### DISCUSSION THREAD

**Discussion by muhammadetman** [Autodesk/maya-usd#2323] [General]
I'm trying to export blendShapes with the skeleton without the mesh but I just can't find a way to do it!

If I select a Mesh and use the exportBlendShapes flag the timeSamples of blendShape node will be added to the def SkelAnimation as a uniform token[] blendShapes = ["blendShape_name"] perfectly fine, however, if I select the root joint and export I don't get the blendShape exported with the skeleton.

I did try and copy the timeSamples exported with the Mesh and copy it manually to a USDA file and it works fine.

My question is, How can I export blendShapes values without the need to export the mesh with it?

Thanks, 
Etman

**Comment by julesarbe** [comment]
Hello ! 
Any news on this subject ? I am trying to achieve a similar thing today and I was wondering if this subject was resolved (it doesn't look like it) 
Thanks !
Jules

**Comment by wallworm** [comment]
We don't currently have support for exporting a blendshape without a mesh. The team will discuss this.
